.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           python -m pip install --upgrade pip
```

Comment:
Hello @tsolakoua, can ```pip install -r requirements.txt``` be used here? As different Sphinx versions are being used for the workflow otherwise.